prado/src/main/java/com/schibsted/spain/prado/imageprovider/VoidImageProvider.kt

```diff
@@ -0,0 +1,15 @@
+package com.schibsted.spain.prado.imageprovider
+
+import android.content.Context
+import android.widget.ImageView
+
+
+class VoidImageProvider : ImageProvider {
+  override fun loadImage(context: Context, imageUrl: String, imageView: ImageView) {
+
```

Comment:
Suggestion: Maybe log a warning into logcat? Just to tell the developer that the invocation is OK and the url is fine, but he's using the void provider.